bin/run-dev-osx.sh

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env bash
+set -e
+
+unset CDPATH
+cd "$( dirname "${BASH_SOURCE[0]}" )/.."
+
+USAGE="
+    Run a development instance of scitran-core\n
+    Also starts mongo, on port 9001 by default\n
+\n
+    Usage:\n
+    \n
+    -C, --config-file <shell-script>: Source a shell script to set environemnt variables\n
+    -I, --no-install: Do not attempt install the application first\n
+    -R, --reload <interval>: Enable live reload, specifying interval in seconds\n
+    -T, --no-testdata: do not bootstrap testdata\n
+    -U, --no-user: do not bootstrap users and groups\n
+"
+
+CONFIG_FILE=""
+BOOTSTRAP_USERS=1
+BOOTSTRAP_TESTDATA=1
+AUTO_RELOAD=0
+INSTALL_APP=1
+
+while [[ "$#" -gt 0 ]]; do
+  key="$1"
+
+  case $key in
+      -C|--config-file)
+      CONFIG_FILE="$1"
+      shift
+      ;;
+      --help)
+      echo -e $USAGE >&2
+      exit 1
+      ;;
+      -I|--no-install)
+      INSTALL_APP=0
+      ;;
+      -R|--reload)
+      AUTO_RELOAD=1
+      AUTO_RELOAD_INTERVAL=$1
+      shift
+      ;;
+      -T|--no-testdata)
+      BOOTSTRAP_TESTDATA=0
+      ;;
+      -U|--no-users)
+      BOOTSTRAP_USERS=0
+      ;;
+      *)
+      echo "Invalid option: $key" >&2
+      echo -e $USAGE >&2
+      exit 1
+      ;;
+  esac
+  shift
+done
+
+set -a
```

Comment:
Let's leave this at `set -o allexport` so that I don't have to look up what `-a` is. Also, you are missing the `+a`.